core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -307,7 +311,11 @@ object ConsoleConsumer extends Logging {
     val partitionArg = if (options.has(partitionIdOpt)) Some(options.valueOf(partitionIdOpt).intValue) else None
     val skipMessageOnError = options.has(skipMessageOnErrorOpt)
     val messageFormatterClass = Class.forName(options.valueOf(messageFormatterOpt))
-    val formatterArgs = CommandLineUtils.parseKeyValueArgs(options.valuesOf(messageFormatterArgOpt).asScala)
+    val formatterArgs =  if (options.has(messageFormatterConfigOpt))
```

Comment:
nit: There is an extra space before `if`.

Reply:
Fixed.